src/query/fetch.c

```diff
@@ -2587,6 +2620,49 @@ fetch_peek_arith (THREAD_ENTRY * thread_p, REGU_VARIABLE * regu_var, VAL_DESCR *
 	}
       break;
 
+    case T_JSON_CONTAINS:
+      if (qdata_json_contains_dbval (peek_left, peek_right, (arithptr->thirdptr == NULL ? NULL : peek_third),
+				     arithptr->value, regu_var->domain) != NO_ERROR)
+	{
+	  goto error;
+	}
+      break;
+
+    case T_JSON_TYPE:
+      if (qdata_json_type_dbval (peek_left, arithptr->value, regu_var->domain) != NO_ERROR)
```

Comment:
That of MySQL returns NULL when NULL argument is given. It also raises an error when a arg is not a valid JSON object.

Reply:
We also raise an error when arg is not a valid JSON object. I have changed the behavior when the argument in NULL to also return NULL.